src/corenlp.cljc

```diff
@@ -286,11 +287,11 @@
 (defmethod dependency-graph DependencyParse [dp]
   "Produce a loom graph from a DependencyParse record."
   (let [[words tags edges] (map #(% dp) [:words :tags :edges])
-        g (apply digraph (map (partial take 2) edges))]
-    (reduce (fn [g [i t]] (add-attr g i :tag t))
-            (reduce (fn [g [i w]] (add-attr g i :word w))
+        g (apply graph/digraph (map (partial take 2) edges))]
+    (reduce (fn [g [i t]] (attr/add-attr g i :tag t))
+            (reduce (fn [g [i w]] (attr/add-attr g i :word w))
                     (reduce (fn [g [gov dep type]]
-                              (add-attr g gov dep :type type)) g edges)
+                              (attr/add-attr g gov dep :type type)) g edges)
```

Comment:
I appreciate that here and elsewhere we are moving to explicit namespace scoping. This is going to keep the code understandable.